static/js/components/About.jsx

```diff
@@ -1,3 +1,26 @@
+/*
+{# Projects extending Skyportal can modify various segments of the About page by
+  writing their own About.jsx.template which extends BaseAbout.jsx.template and
+  overwrites the Jinja template blocks they wish to customize. Here, we just
+  render the base template as is. This way, just the specific parts that need to
+  be customized/appended to can be overwritten without having to overwrite the
+  entire About.jsx component.
+
+  Note that the base template uses comments to comment out Jinja-specific syntax
+  like the block declarations in order to make the template double as a valid
+  JSX file. Thus, further care should be taken when extending the template in
+  order to keep those comment characters from commenting out the blocks you
+  choose to overwrite. For example, you should place "*\/}" (without the escape
+  backslash) after {% block %} declarations within the JSX component render()
+  segment, and "{/*" immediately before the {% endblock %} declarations within
+  the render().
+
+  See https://jinja.palletsprojects.com/en/2.11.x/templates/#template-inheritance
+  for documentation on usage of Jinja templates.
+#}
+*/
+
+// {/* {% block imports -%} */}
```

Comment:
Can we stick to the `//` syntax consistently everywhere?  That would also make it easy to explain.

Reply:
I wish, but within the HTML parts the `//` doesn't actually make a comment so I don't think we can use it for the blocks in the return() statement.